po/es.po

```diff
@@ -3275,7 +3275,7 @@ msgid ""
 "starting index, meaning that `&a[0..a.len()]` and `&a[..a.len()]` are "
 "identical."
 msgstr ""
-"Si el slice comienza en el índice 0, la sintaxis de intervalo de Rust nos "
+"Si el slice comienza en el índice 0, la sintaxis de rango de Rust nos "
```

Comment:
I don't agree with this suggestion. "rango" can mean rank, which is very misleading here. "intervalo" is unambiguous

![image](https://github.com/google/comprehensive-rust/assets/15252789/29837d4d-665d-4c79-a6a7-fba8ef2a102a)

Reply:
Yesterday, @ph4un00b and I discussed this. "Intervalo" isn't a poor translation, it's akin to "time interval" and offers a more precise definition, especially for those with a mathematical background. However, for the majority of people, using terms like "rangos" when referring to range types is more commonplace.

![image](https://github.com/google/comprehensive-rust/assets/19656993/cec70b7f-00e9-4840-bfad-a17860ed13fd)

Some examples of that:
[Python](https://elpythonista.com/range-enumerate#:~:text=valor%20de%20parada.-,%C2%BFC%C3%B3mo%20crear%20rangos%20en%20Python%3F,-La%20sint%C3%A1xis%20para)
[Ruby](http://rubytutorial.wikidot.com/rangos)
[Swift](https://www.lafactoriaapple.com/swift/bucles/rangos.php)

Even when we're discussing Rust, we use the term "rango"
[FreecodeCamp Tutorial](https://www.freecodecamp.org/espanol/news/rust-en-replit/#:~:text=la%20sintaxis%20de%20rango%20de%20Rust)

Reply:
I like it either way. Intervalo and rango works in this context for me.

Reply:
@Phosphorus-M, can you take out the translation here for now? Then try to reach a consensus with @JaviSorribes and the other reviewers and members of the Spanish Rust community.

Basically, I would love to merge this since it removes the weird-looking `_foo_` words from the TOC. So if you could take out the translation here, then we can make forward progress. The translation can still go in, it should just be discussed in a smaller and more targeted PR.

Reply:
Hi all, please come to a conclusion here so we can merge this — I know it's hard to edit a large body of text like this, so I suggest taking out the change for now.